rasterio/mask.py

```diff
@@ -3,7 +3,7 @@
 import logging
 import warnings
 
-import numpy
+import numpy as np
```

Comment:
This change is not relevant to this PR. I recommend leaving it as is and discussing it separately.

Reply:
I agree it is not directly relevant to this PR, however, since I updated the only line where numpy is used, I took the opportunity to standardize the import as well. I believe this standardization has already been discussed (#727) and accepted (#738).

Reply:
Thanks for pointing out the issue - the change should be fine in that case. I personally would have gone for the first option of just `import numpy`, but better to be consistent for now.